digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -293,11 +294,19 @@ protected TaskResult callExecutor(Path projectPath, String type, TaskRequest mer
                 .build();
 
         DefaultSecretProvider secretProvider = new DefaultSecretProvider(
-                secretContext, secretAccessPolicy, grants, operatorSecretFilter, secretStore);
+                secretContext, secretAccessPolicy, grants, (key) -> declaredSecretKeys.contains(key), secretStore);
+
+        PrivilegedVariables privilegedVariables = GrantedPrivilegedVariables.build(
+                mergedRequest.getConfig().getNestedOrGetEmpty("_env"),
```

Comment:
Is `_env` a valid configuration key for all operators? It does not mean "environment" as in system environment variables?